airflow/www/views.py

```diff
@@ -1185,16 +1186,16 @@ def tree(self, session=None):
         else:
             base_date = dag.latest_execution_date or timezone.utcnow()
 
-        DR = models.DagRun
-        dag_runs = (
-            session.query(DR)
-            .filter(
-                DR.dag_id == dag.dag_id,
-                DR.execution_date <= base_date)
-            .order_by(DR.execution_date.desc())
-            .limit(num_runs)
-            .all()
-        )
+        with create_session() as session:
```

Comment:
We've already got a `session` object in scope here.

Reply:
not anymore, just did remove that